emails/models.py

```diff
@@ -517,6 +529,9 @@ def delete(self, *args, **kwargs):
         profile.num_address_deleted += 1
         profile.num_email_forwarded_in_deleted_address += self.num_forwarded
         profile.num_email_blocked_in_deleted_address += self.num_blocked
+        profile.num_level_one_trackers_blocked_in_deleted_address += (
+            self.num_level_one_trackers_blocked
+        )
```

Comment:
Need to expect either to be none:

```suggestion
        profile.num_level_one_trackers_blocked_in_deleted_address = (
            (profile.num_level_one_trackers_blocked_in_deleted_address or 0) +
            (self.num_level_one_trackers_blocked or 0)
        )
```